packages/monaco/src/browser/monaco-frontend-module.ts

```diff
@@ -162,16 +160,6 @@ export function createMonacoConfigurationService(container: interfaces.Container
         return proxy;
     };
 
-    const initFromConfiguration = debounce(() => {
```

Comment:
it is important to keep this code to make sure that all default context key values are loaded properly, i.e. see https://github.com/theia-ide/vscode/blob/b0b47123a5da83d42c2675f2bfff5bb9f1b2673c/src/vs/platform/contextkey/browser/contextKeyService.ts#L103-L108

Reply:
although let's try without it, I cannot reproduce excessive default events anymore, there seems to be only one startup